[MIG] public_budget & sipreco *: Migration to 15.0

pablohmontenegro opened the pull request.

pablohmontenegro committed:
[MIG] public_budget: Migration to 15.0

Commit:
[REM] deprecate check functionality
Borramos distintas funcionalidades de cheques que cambiaron mucho con el nuevo release. Las borramos porque en 2022 no están usando cheques

roboadhoc commented:
[Pull request status dashboard](https://runbot.adhoc.com.ar/ingadhoc/odoo-public-administration/pull/438).

Commit:
[CHANGES]modificaciones varias

nicomacr committed:
[MIG] public_budget_subscription: Migration to 15.0

nicomacr committed:
[MIG] public_budget_tax_settlement: Migration to 15.0

nicomacr committed:
[MIG] sipreco_payment_line: Migration to 15.0

nicomacr committed:
[MIG] sipreco_position_helper: Migration to 15.0

nicomacr committed:
[MIG] sipreco_project: Migration to 15.0

nicomacr committed:
[MIG] sipreco_subsidy_management: Migration to 15.0

nicomacr committed:
[MIG] sipreco_purchase: Migration to 15.0

pablohmontenegro committed:
[MIG]public_budget:corregir cta contable, mvto fondos

pablohmontenegro committed:
[FIX]public_budget_tax_settlement: field enable to pay on moves

pablohmontenegro committed:
[FIX]public_budget: permitir guardar pago a proveedor con línea de pago

Faltaba setear date en account.payment como no storeado porque sino cuando guardo y me crea el asiento al ser date un campo almacenado en el account.payment y account.move al venir seteado en false tira error entonces al crear el asiento con date en false, para que no suceda eso, el campo date lo pongo como no storeado en account.payment y se soluciona

pablohmontenegro committed:
[FIX] public_budget: permitir creación masiva de comprobantes

pablohmontenegro committed:
[FIX] public_budget: show partner name on invoices

pablohmontenegro committed:
[FIX] public_budget: crear una orden de pago por cada factura en una transacción
Ticket: 56055

nicomacr commented:
@roboadhoc r+ rebase-ff

roboadhoc commented:
Merge method set to rebase and fast-forward
